docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -153,8 +153,13 @@ Will return a JSON object of the form:
   "Images" : [
     {
       "Candidate": "123456",
+      "PSCID": "MTL001",
       "Visit": "V1",
+      "Visit_date": "2016-08-09",
```

Comment:
This is ambiguous and should be described in the text. Which date is it? The date from the scan header? The date from the session? The date from the mri_parameter_form? Etc...

Reply:
this Visit date would be NULL for phantoms (always) and for candidates whose MRI visit has not started yet; is it worth specifying this?